debian/rules

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/make -f
+
+SERVICE = neofs-node
+VERSION = $(shell make --no-print-directory version | sed "s/^v//" | sed -r "s/(.*)-(g[a-fA-F0-9]{6,8})(.*)/\1\3~\2/" | sed "s/-/~/")
+DISTRIBUTION = $(shell sed -n "s/^VERSION_CODENAME=//p" /etc/os-release)
+PACKAGEVERSION = $(VERSION)-$(DISTRIBUTION)
+
+%:
+	dh $@ --with bash-completion
+
+override_dh_auto_test:
+
+override_dh_auto_install:
+		dh_auto_install
+		install -D -m 0750 -t debian/$(SERVICE)/bin bin/*
+		bin/neofs-adm gendoc -t man man/
+		bin/neofs-adm completion bash > debian/neofs-adm.bash-completion
+		bin/neofs-cli gendoc -t man man/
+		bin/neofs-cli completion bash > debian/neofs-cli.bash-completion
+		install -m 0755 -d debian/$(SERVICE)/usr/share/fish/completions
+		bin/neofs-cli completion fish > debian/$(SERVICE)/usr/share/fish/completions/neofs-node.fish
+		install -m 0755 -d debian/$(SERVICE)/usr/share/zsh/vendor-completions
+		bin/neofs-cli completion zsh > debian/$(SERVICE)/usr/share/zsh/vendor-completions/_neofs-node
+		wget https://github.com/nspcc-dev/neofs-locode-db/releases/download/v0.2.1/locode_db.tar.gz
```

Comment:
Just noticed. `wget` shouldn't be used like this, we can have multiple source files for the package, although it probably is not as easy as with RPM specs.